Tests/NIOOpenSSLTests/SSLCertificateTest.swift

```diff
@@ -251,8 +251,8 @@ class SSLCertificateTest: XCTestCase {
         do {
             _ = try OpenSSLCertificate(file: "/nonexistent/path", format: .pem)
             XCTFail("Did not throw")
-        } catch NIOOpenSSLError.noSuchFilesystemObject {
-            // ok
+        } catch let error as IOError {
+            XCTAssertEqual(error.errnoCode, ENOENT)
```

Comment:
All of these test changes are a change in the thrown error, but they're semver patch because these functions never used to throw in this circumstance at all: they crashed. The error thrown before this patch was added in #23.